CHANGELOG.md

```diff
@@ -2,14 +2,16 @@
 ## Common changes for all artifacts
 
 ## stream-chat-android
-Remove `ChatClient` and `ChatDomain` as `ChatUI`'s dependecies
+Remove `ChatClient` and `ChatDomain` as `ChatUI`'s dependencies
 
 ## stream-chat-android-client
-- Depecrate `User::unreadCount` property, replace with `User::totalUnreadCount`
+- Deprecate `User::unreadCount` property, replace with `User::totalUnreadCount`
+- Add MarkAllReadEvent
 
 ## stream-chat-android-offline
 - Update `totalUnreadCount` when user is connected
 - Update `channelUnreadCount` when user is connected
+- Add support for marking all channels as read for the current user
```

Comment:
Would be nice to mention how to do this more precisely, i.e. class/method name

Reply:
Will do. Wasn't sure how long or explicit changelog comments were intended to be.